docs/openapi.json

```diff
@@ -94,40 +115,19 @@
         }
       }
     },
-    "/contracts/v1/documents" : {
-      "post" : {
-        "tags" : [ "documents" ],
-        "summary" : "upload a Document for a contract at Baloise.",
-        "description" : "Please note that this operation needs to be called per document for a contract to be uploaded. The response contains a handle to the document. This handle should be provided with the contract to be uploaded via uploadContract",
-        "operationId" : "uploadDocument",
-        "requestBody" : {
-          "description" : "a documnent that is part of a contract",
-          "content" : {
-            "application/json" : {
-              "schema" : {
-                "$ref" : "#/components/schemas/Document"
-              }
-            }
-          },
-          "required" : true
-        },
+    "/contracts/v1/version" : {
+      "get" : {
+        "tags" : [ "version" ],
+        "summary" : "Callable way of retrieving current version",
+        "description" : "Callable way (compared to analyzing the URL of the endpoint) of retrieving current version. As well good for testing purposes as operation is get only",
```

Comment:
Would rephrase to "Endpoint to retrieve the current API version. Can be compared to the URI version. Can be used for testing purposes."